spec/bundler/fetcher/compact_index_spec.rb

```diff
@@ -35,11 +35,8 @@
       end
 
       context "when OpenSSL is not available" do
-        before do
-          allow(compact_index).to receive(:require).with("openssl").and_raise(LoadError)
-        end
-
         it "returns true" do
+          allow(Bundler::SharedHelpers).to receive(:require).with("openssl").and_raise(LoadError)
```

Comment:
Does this need to be moved into the `it` block for any specific reason?

Reply:
yeh, it's rspec specific, unless its `suibject` it needs to be in the `it` block.

Reply:
Can you clarify your answer @frezbo? I didn't understand it, and I wonder the same thing @colby-swandale wonders.